install_requirements.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
can we test that script somehow?

Reply:
I don't feel comfortable writing a test for that script. I have not enough experiences and I am not sure what should be tested here and how to implemtent the tests in the environment.

Basically I copied the code from [how_to_contribute.rst](https://github.com/blue-yonder/tsfresh/blob/main/docs/text/how_to_contribute.rst)

However, I have the following ideas:
- Check whether the isntall script and the infos in how_to_contribust.rst match
- Check, whether all "*requirements.txt" files are listed
- Make sure, all listed files meet the regex "*requirements.txt" and that there is only one additional line (`pip install -e .`)

You can point me in the right direction implementation-wise.